src/main/java/com/box/sdk/CreateUserParams.java

```diff
@@ -28,7 +29,7 @@ public class CreateUserParams {
      *
      * @return true if the new user will be able to see other enterprise users in their contact list; otherwise false.
      */
-    public boolean getCanSeeManagedUsers() {
+    public Boolean getCanSeeManagedUsers() {
```

Comment:
This is a breaking change in the API we have to mark it in the PR title

Reply:
I consider it as a small breaking change (Usually clients won't use getters for an object used only to make a request). Would adding `!` not cause bumping major version ?